packages/cactus-plugin-ledger-connector-polkadot/package.json

```diff
@@ -0,0 +1,114 @@
+{
+  "name": "@hyperledger/cactus-plugin-ledger-connector-polkadot",
```

Comment:
- Add this package to tsconfig references (https://github.com/hyperledger/cacti/blob/main/tsconfig.json#L3) to include it in build
- Add this package to CI (like that - https://github.com/hyperledger/cacti/blob/main/.github/workflows/ci.yaml#L200, but set JEST_TEST_RUNNER_DISABLED to false if this package use tape tests only.

Reply:
added the package to tsconfig, second not needed as tests are now in jest

Reply:
@AnmolBansalDEV CI entry (P2) is still needed, but with `JEST_TEST_RUNNER_DISABLED` set to false and `TAPE_TEST_RUNNER_DISABLED` set to `true`, something like the following:

```
  cactus-plugin-ledger-connector-polkadot:
    continue-on-error: false
    env:
      FULL_BUILD_DISABLED: true
      JEST_TEST_PATTERN: packages/cactus-plugin-ledger-connector-polkadot/src/test/typescript/(unit|integration|benchmark)/.*/*.test.ts
      JEST_TEST_RUNNER_DISABLED: false
      TAPE_TEST_RUNNER_DISABLED: true
    needs: build-dev
    runs-on: ubuntu-20.04
    steps:
      - name: Use Node.js ${{ env.NODEJS_VERSION }}
        uses: actions/setup-node@v3.6.0
        with:
          node-version: ${{ env.NODEJS_VERSION }}
      - uses: actions/checkout@v3.5.2

      - id: yarn-cache
        name: Restore Yarn Cache
        uses: actions/cache@v3.3.1
        with:
          key: ${{ runner.os }}-yarn-${{ hashFiles('./yarn.lock') }}
          path: ./.yarn/
          restore-keys: |
            ${{ runner.os }}-yarn-${{ hashFiles('./yarn.lock') }}
      - run: ./tools/ci.sh
```

Reply:
Also, when you're done with requested changes please hit `re-request review` circle button, next to reviewer avatar in top-right pane to notify us that there's something to see here :)

Reply:
sure, updated. And requested your kind review.